### ISSUE TITLE
Ubuntu Suru icons

### ISSUE THREAD

**Issue by mivoligo** [Xaviju/inkscape-open-symbols#1]
Hi,
Can you add Ubuntu Suru icons to the collection, please? These icons are used in the Ubuntu for phones at the moment. Link to the source files: http://bazaar.launchpad.net/~ubuntu-art-pkg/ubuntu-themes/trunk/files/head:/suru-icons/
Many thanks

**Comment by Xaviju** [comment]
Hi @mivoligo 
They look really nice, but sadly they are not clean (look at the difference with the google material design code icons), they need to be cleaned up in order to be easily converted to symbols and this will take a lot of time.
I can not afford enough time to do that right now but if anyone is willing to do this I will be happy to help, step by step, on how to create the first one.
In any case, I will add this enhancement to our icon sets list.

Thanks for the reference, the icons are awesome.

**Comment by mivoligo** [comment]
Thanks for quick answer. Let me know how to "clean them up". I can give it a shot in my spare time.

**Comment by Xaviju** [comment]
I have written [an article](https://medium.com/@xaviju/creating-your-own-symbol-library-in-inkscape-0-91-and-make-your-front-end-developer-you-338588137aaf) that you might find useful to start.
There is an example on the root folder called a "[template.svg](https://github.com/Xaviju/inkscape-open-symbols/blob/master/template.svg)" where you can see how it should look like.

If you make the first one, I can test it and check if its correct.

**Comment by mivoligo** [comment]
OK, so the most important thing is that one icon should be made of paths, right? Also I found this video: https://www.youtube.com/watch?v=jdaGB0zCdHw Is it fine to put all together like in the video, when the icons are cleaned?

**Comment by Xaviju** [comment]
Nice video.
Let's do some experiment, create the icons from the UI and send a PR with the result and lets check how it looks. I'm not sure that it will be as useful as if we make it manually but might be enough for the beggining. If necesary, I'll clean the inkscape and sodipodi extra info.
The idea behind the cleaning is that the same icons, just its code copied and pasted, could be useful for front-end development as well.

**Comment by mivoligo** [comment]
I'll try to do that in the evening and let you know.

**Comment by Xaviju** [comment]
nice :+1:

**Comment by mivoligo** [comment]
I've made a test set with 10 icons. I didn't change anything in them, just did like it was shown on that video. Take a look at PR. I'm afraid it does not look good.
BTW I'm rather new in using github so please excuse if I messed something up ;)

**Comment by Xaviju** [comment]
Hi @mivoligo I just saw the code and looks a little bit complicated, but I need to test it more to be sure. I made a test symbol library with my own created symbols from the UI ([see the update!](https://github.com/Xaviju/inkscape-open-symbols/wiki/Humble-UI)) and something can be done but not much. The SVG code its not perfect, but enough for designers.
I'll be out until sunday but I will test it as soon as possible when I come back.

In any case, if it works correctly, those icons can be published so we can start to use them and I will clean them out later on. :smiley: 

Thanks!
